modules/fluid_structure_interaction/test/tests/fsi_acoustics/1D_fluid_only/1D_fluid_only.i

```diff
@@ -0,0 +1,110 @@
+# Test for `AcousticInertia` and `Diffusion` kernels with only the fluid domain. The
+#  domain is 1D with length 1m and is subjected to an initial condition composed of
+#  a combination of sine waves. Fluid pressure is recorded at the midpoint of the
+# domain. The recorded fluid pressure should match with analytical results. Because
+#  this implementation is equivalent to solving a 1D wave equation, analytical results
+#  exist.
```

Comment:
```suggestion
# Test for `AcousticInertia` and `Diffusion` kernels with only the fluid domain. The
# domain is 1D with length 1m and is subjected to an initial condition composed of
# a combination of sine waves. Fluid pressure is recorded at the midpoint of the
# domain. The recorded fluid pressure should match with analytical results. Because
# this implementation is equivalent to solving a 1D wave equation, analytical results
# exist.
```